tests/create_appliance.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+
+from swsscommon import swsscommon
+from dash_api.appliance_pb2 import *
```

Comment:
>from [](http://example.com/codeflow?start=0&length=4)

Could you add a comment section at the file head? At least explain this script intention, and usage of each command line arg.

Reply:
Fixed